k8s/nginx-ingress/prod-api-ingress-resource.yaml

```diff
@@ -56,16 +56,6 @@ metadata:
 
 spec:
   rules:
-    - host: airqalibrate.airqo.net
-      http:
```

Comment:
This was duplicate.
It already exists in the `k8s/nginx-ingress/prod-platform-ingress-resource.yaml` file